src/core_modules/capture-core/components/WidgetEventSchedule/AccessVerification/NoAccess.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { type ComponentType } from 'react';
+import { withStyles, type WithStyles } from '@material-ui/core';
+import i18n from '@dhis2/d2-i18n';
+import { Button } from '@dhis2/ui';
+import { NoWriteAccessMessage } from '../../NoWriteAccessMessage';
+
+const styles: Readonly<any> = {
+    buttonRow: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        paddingTop: 10,
+        marginLeft: '-8px',
+    },
+    buttonContainer: {
+        paddingRight: 16,
```

Comment:
Do not remove the use of theme:

```suggestion
        paddingRight: theme.spacing.unit * 2,
```